src/tls13crypto.rs

```diff
@@ -310,6 +315,7 @@ impl SignatureScheme {
 }
 
 /// Sign the `input` with the provided RSA key.
+#[hax_lib_macros::pv_constructor]
```

Comment:
To make these a little cleaner and easier to read, add a `use hax_lib_macros::*` import.